join_skip.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Join SKIP
+layout: post
+---
+
+## Join Us: Become a SKIP Member!
+
+Are you eager to be part of a dynamic and impactful student association? At SKIP, we welcome students who are enthusiastic about enhancing the academic and social experience of their peers. Whether you're interested in taking a leadership role as a board member or providing valuable support as a regular member, there's a place for you in our community.
+
+1. **Regular Members:**
+As a regular member, you can actively contribute to the success of SKIP. Your participation in events and feedback are highly valued. Additionally, you'll enjoy perks like priority access to events and discounts. If you're ready to dive in and be an essential part of the SKIP experience, apply today by clicking
+
+2. **Board Members:**
+If you're passionate about event planning, leadership, and making a significant impact, consider joining our board. Board members have a crucial role in shaping the direction of SKIP. Please note that board member applications are individually reviewed. If selected, you will be scheduled for an onboarding process to help you transition into your leadership role effectively. To apply for a board position, click [LINK](link_goes_here)!
+
+Should you have any questions or require further information, please don't hesitate to reach out to us at skip@gota.gu.se. We look forward to welcoming you as part of the SKIP family!
```

Comment:
I'd **suggest** to use: [*skip@gota.gu.se*](mailto:skip@gota.gu.se).

Reply:
Are you referring to the email being a mailto: link, or do you want it in italics?

Reply:
Oh, both, without the rendering:
```md
[*skip@gota.gu.se*](mailto:skip@gota.gu.se)
```

Reply:
I see it's correctly implemented now!